torchok/models/backbones/__init__.py

```diff
@@ -1,66 +1,4 @@
 from torchok.models.backbones.base_backbone import BaseBackbone
-
-from torchok.models.backbones.resnet import (
-    create_resnet,
-    resnet18,
-    resnet34,
-    resnet50,
-    resnet101,
-    resnet152,
-    seresnet18,
-    seresnet34,
-    seresnet50,
-    seresnet101,
-    seresnet152,
-)
-from torchok.models.backbones.hrnet import (
-    create_hrnet,
-    hrnet_w18_small,
-    hrnet_w18_small_v2,
-    hrnet_w18,
-    hrnet_w30,
-    hrnet_w32,
-    hrnet_w40,
-    hrnet_w44,
-    hrnet_w48,
-    hrnet_w64,
-)
-from torchok.models.backbones.swin import (
-    swinv2_tiny_window16_256,
-    swinv2_tiny_window8_256,
-    swinv2_small_window16_256,
-    swinv2_small_window8_256,
-    swinv2_base_window16_256,
-    swinv2_base_window8_256,
-    swinv2_base_window12_192_22k,
-    swinv2_base_window12to16_192to256_22kft1k,
-    swinv2_base_window12to24_192to384_22kft1k,
-    swinv2_large_window12_192_22k,
-    swinv2_large_window12to16_192to256_22kft1k,
-    swinv2_large_window12to24_192to384_22kft1k
-)
-
-__all__ = [
-    'ResNet',
-    'create_resnet',
-    'resnet18',
-    'resnet34',
-    'resnet50',
-    'resnet101',
-    'resnet152',
-    'seresnet18',
-    'seresnet34',
-    'seresnet50',
-    'seresnet101',
-    'seresnet152',
-    'create_hrnet',
-    'hrnet_w18_small',
-    'hrnet_w18_small_v2',
-    'hrnet_w18',
-    'hrnet_w30',
-    'hrnet_w32',
-    'hrnet_w40',
-    'hrnet_w44',
-    'hrnet_w48',
-    'hrnet_w64',
-]
+from torchok.models.backbones import resnet
```

Comment:
I still think that if we write this line we add timm modules in documentation. If it will be like i think, will you fix it by yorself?

Reply:
Yes, I will